python-311/tomllib_w.py

```diff
@@ -0,0 +1,39 @@
+# tomllib_w.py
+
+from datetime import date
+
+
+def dumps(toml_dict, table=""):
+    def tables_at_end(item):
+        _, value = item
+        return isinstance(value, dict)
+
+    document = []
+    for key, value in sorted(toml_dict.items(), key=tables_at_end):
+        match value:
```

Comment:
Oh, that's interesting.

It runs smoothly for me. I suppose it's because I've installed `flake8` on Python 3.10, and it uses the underlying Python to parse the AST?

```console
$ flake8 --version
4.0.1 (mccabe: 0.6.1, pycodestyle: 2.8.0, pyflakes: 2.4.0) CPython 3.10.2 on Linux
```